test/units/res.clearCookie.test.ts

```diff
@@ -1,9 +1,21 @@
 import { opine } from "../../mod.ts";
-import { superdeno } from "../deps.ts";
+import { superdeno, SuperDenoResponse, expect } from "../deps.ts";
```

Comment:
If `SuperDenoResponse` is indeed required (would hope that we get the types for free - shout it not and can investigate in the SuperDeno repo!) we should import using the `import type` syntax:

```suggestion
import { superdeno, expect } from "../deps.ts";
import { SuperDenoResponse } from "../deps.ts";
```